gradle/spotbugs-exclude.xml

```diff
@@ -49,6 +49,7 @@ For a detailed description of spotbugs bug categories, see https://spotbugs.read
              some categories of bug reports when using Scala, since spotbugs generates huge
              numbers of false positives in some of these categories when examining Scala code.
 
+            SA_FIELD_SELF_COMPARISON: This method compares a field with itself, and may indicate a typo or a logic error.
```

Comment:
Thanks for the investigation. I agree it should be false positive. Could we add the spotbugs github issue link here to indicate this should be removed after issue fixed?

Reply:
Thanks @showuon , great idea, github issue link added!